builtin/logical/pki/path_tidy.go

```diff
@@ -88,6 +63,36 @@ func pathTidyStatus(b *backend) *framework.Path {
 	}
 }
 
+func pathConfigAutoTidy(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/auto-tidy",
+		Fields: addTidyFields(map[string]*framework.FieldSchema{
+			"enabled": {
+				Type:        framework.TypeBool,
+				Description: `Set to true to enable automatic tidy operations.`,
+			},
+			"interval_duration": {
+				Type:        framework.TypeDurationSecond,
+				Description: `Interval at which to run an auto-tidy operation. This is the time between tidy invocations (after one finishes to the start of the next). Running a manual tidy will reset this duration.`,
+				Default:     43200, // 32h, but TypeDurationSecond currently requires the default to be an int.
```

Comment:
43200s = 12h. Is 32h default a typo?

Reply:
Ah, good eyes. Yes, a typo. Note though that auto-tidy isn't enabled by default.